vars/defaults.yml

```diff
@@ -5,6 +5,7 @@
 ###############################################################################
 
 # # common
+# common_timezone: 'Etc/UTC' (required)
```

Comment:
No don't think so its already currently the default. I will set it as default then.